src/Components/Facility/Consultations/GlasgowTables.tsx

```diff
@@ -0,0 +1,190 @@
+import { result } from "lodash";
+import moment from "moment";
+import { useCallback, useState } from "react";
+import { useDispatch } from "react-redux";
+import { statusType, useAbortableEffect } from "../../../Common/utils";
+import { dailyRoundsAnalyse } from "../../../Redux/actions";
+
+export const GlasgowTables = (props: any) => {
+  const { facilityId, patientId, consultationId } = props;
+  const dispatch: any = useDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const [offset, setOffset] = useState(0);
+  const [results, setResults] = useState({});
+
+  const EYE_OPEN_SCALE = [
+    { value: 4, text: "Spontaneous" },
+    { value: 3, text: "To Speech" },
+    { value: 2, text: "Pain" },
+    { value: 1, text: "None" },
+  ];
+
+  const VERBAL_RESPONSE_SCALE = [
+    { value: 5, text: "Oriented/Coos/Babbles" },
+    { value: 4, text: "Confused/Irritable" },
+    { value: 3, text: "Inappropriate words/Cry to Pain" },
+    { value: 2, text: "Incomprehensible words/Moans to pain" },
+    { value: 1, text: "None" },
+  ];
+
+  const MOTOR_RESPONSE_SCALE = [
+    { value: 6, text: "Obeying commands" },
+    { value: 5, text: "Moves to localised pain" },
+    { value: 4, text: "Flexion withdrawal from pain" },
+    { value: 3, text: "Abnormal Flexion(decorticate)" },
```

Comment:
can you move these constants in `constants.ts` where already many constants are defined.